app/src/main/res/layout/activity_main.xml

```diff
@@ -1,125 +1,151 @@
-<?xml version="1.0" encoding="utf-8"?>
+ <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:alpha="0.7"
-    android:background="@drawable/background"
+    android:background="@color/ligthBeige"
     tools:context=".MainActivity">
 
     <TextView
         android:id="@+id/title"
-        android:layout_width="1283dp"
+        android:layout_width="0dp"
         android:layout_height="0dp"
         android:layout_marginTop="1dp"
-        android:layout_marginBottom="71dp"
-        android:alpha="0.8"
-        android:background="#F2F0D1"
+        android:layout_marginBottom="658dp"
+        android:background="@color/darkBeige"
         android:fontFamily="@font/share_regular"
         android:paddingTop="15dp"
         android:shadowColor="@color/black"
         android:text="Drink Master"
         android:textAlignment="center"
+
```

Comment:
watch fort the space. Don't make an emplty line to just continue working on the same object